sapl/painel/views.py

```diff
@@ -32,9 +32,19 @@
 class CronometroPainelCrud(CrudAux):
     model = Cronometro
 
+    class ListView(CrudAux.ListView):
+        template_name = "painel/cronometro_list.html"
+        
     class BaseMixin(CrudAux.BaseMixin):
         form_class = CronometroForm
 
+def ordena_cronometro(request):
+    ordem = request.POST.getlist("ordem", [])
+    for index, o in enumerate(ordem):
```

Comment:
Vc pode fazer assim: 
```suggestion
   for index, o in enumerate(ordem, 1):
```

Assim, não precisa colocar `c.ordenacao = index` (não precisa somar o 1).

Reply:
Solução implementada.